core/src/worker/workflow/machines/workflow_machines.rs

```diff
@@ -781,7 +781,10 @@ impl WorkflowMachines {
                 Ok(EventHandlingOutcome::Normal)
             };
         }
-        if event.event_type() == EventType::Unspecified || event.attributes.is_none() {
+        if event.event_type() == EventType::Unspecified
+            || event.event_type() == EventType::WorkflowExecutionUpdateRequested
+            || event.attributes.is_none()
+        {
```

Comment:
There are two manual changes to `sdk-core` in this PR. This is one of them. All other changes were made by updating the protos subtree.